server/main.ml

```diff
@@ -1,20 +1,33 @@
 module Server = Oca_server.Server.Make (Backend)
 
-let main debug workdir = Lwt_main.run (Server.main ~debug ~workdir)
+let main debug workdir cap_file = Lwt_main.run (Server.main ~debug ~cap_file ~workdir)
 
-let term =
-  let ($) = Cmdliner.Term.($) in
-  Cmdliner.Term.pure main $
-  Cmdliner.Arg.(value & flag & info ["debug"]) $
-  Cmdliner.Arg.(required & pos 0 (some string) None & info ~docv:"WORKDIR" [])
+(* Command-line parsing *)
+
+open Cmdliner
```

Comment:
```suggestion
module Arg = Cmdliner.Arg
module Term = Cmdliner.Term
```
I prefer to have as little `open` as possible. I regard it as bad practice for myself.
I use it only for local opens (i try not to abuse them as well), for stdlibs and `Infix` submodules.

Reply:
Ok, I spend to much time on the other ocurrent projects where that's the style.